resources/references/adr/2020-08-12-implement-app-system-inside-platform.md

```diff
@@ -56,7 +56,7 @@ Additionally, the storefront theme files (JS and CSS) only need to be accessed d
 
 * [Custom Fields](https://docs.shopware.com/en/shopware-platform-dev-en/app-system-guide/app-base-guide?category=shopware-platform-dev-en/app-system-guide#custom-fields): An app can register it's own custom fields sets, that are displayed along the other custom fields inside the administration.
 
-* [Storefront Customizations](https://docs.shopware.com/en/shopware-platform-dev-en/app-system-guide/app-examples-and-tutorials/create-own-theme?category=shopware-platform-dev-en/app-system-guide/app-examples-and-tutorials): An app should be able to customize the storefront in the same way a plugin does. This includes the theme system, custom twig templates and custom JS and CSS.
+* [Storefront Customizations](/docs/guides/plugins/apps/storefront/): An app should be able to customize the storefront in the same way a plugin does. This includes the theme system, custom twig templates and custom JS and CSS.
```

Comment:
Note: Add full path / url in next platform MR

Reply:
https://shopware.atlassian.net/jira/software/c/projects/DX/boards/336/backlog?selectedIssue=DX-497